jx_sqlite/setop_table.py

```diff
@@ -259,7 +258,7 @@ def _accumulate_nested(rows, row, nested_doc_details, parent_doc_id, parent_id_c
                             if value == '':
                                 continue
 
-                            relative_path = relative_field(concat_field(c.push_name, c.push_child), curr_nested_path)
+                            relative_path = relative_field(join_field([c.push_name]+split_field(c.push_child)), curr_nested_path)
                             if relative_path == ".":
```

Comment:
plus,`join_field([c.push_name]+split_field(c.push_child)` is technically wrong, even if it works right now: `c.push_name` can be a dot-delimited property path; wrapping it in an array singleton `[c.push_name]` will change the meaning of those dots.